sslcert-review.py

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""SSL Grader"""
+__author__ = "Kevin Amorin"
+__copyright__ = "Copyright 2020"
+__license__ = "GPL"
+__version__ = "1.0.1"
+
+import sys, os
+from shodan import Shodan
+from pprint import pprint, pformat
+import pickle
+from OpenSSL import crypto
+from datetime import datetime
+import certifi
+import pem
+import logging
+import argparse
+from prettytable import PrettyTable
+import csv
+import censys.certificates
+import censys.ipv4
+
+ROOT_STORE = None
+
+def log(s, type="DEBUG"):
+    """ log wrapper """
+    levels = {"DEBUG": 10, "INFO": 20, "WARNING": 30, "ERROR": 40, "CRITICAL": 50}
+    logging.log(levels[type], s)
+
+
+def extract_altname(server_crt):
+    """ Helper: parse PEM formated certificate chain list for v3 extention alt-names
+
+        :param server_crt: list of PEM certs in UTF-8 string format
+        :type server_crt: list
+    """
+    x509cert = crypto.load_certificate(crypto.FILETYPE_PEM, server_crt)
+    san = ""
+    ext_count = x509cert.get_extension_count()
+    for i in range(0, ext_count):
+        ext = x509cert.get_extension(i)
+        if "subjectAltName" in str(ext.get_short_name()):
+            san = ext.__str__().replace("DNS", "").replace(":", "").split(", ")
+
+    return san
+
+
+def load_root_ca_list(debug=False):
+    """ load all certificates found in openssl cert.pem (via certifi.where())
+        
+        :return: returns X509store obj loaded with trusted Cert.  
+        :rtype: X509store
+    """
+    store = None
```

Comment:
you can remove this declaration, `return store` at the end of the `with` and `return None` at the end of this function